src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
+<dt markdown="1">**Tree View of Memory**</dt>
+<dd>Table Tree view of outstanding memory events e.g., User requested snapshots, Automatic snapshots, Analysis of snapshots, and Memory Allocation monitoring.</dd>
+<dt markdown="1">**Memory View**</dt>
+<dd>Display either the contents of the analysis, snapshot or monitoring based on the currently selected row in the Table Tree view.</dd>
+</dl>
+
+## Memory Controls
+
+The top line of the memory profile page has a number of buttons and dropdowns to control how memory charts are displayed.
+
+![Screenshot of a memory controls]({% asset tools/devtools/memory_controls.png @path %}){:width="100%"}
+
+<dl markdown="1">
+<dt markdown="1">**Pause**</dt>
+<dd>Pause the memory overview chart to allow inspecting
+    the currently plotted data. Incoming memory data is still received;
+    notice the Range selector continues to grow to the right.</dd>
+<dt markdown="1">**Resume**</dt>
+<dd>The memory overview chart is live and displaying the
+    current time and the latest memory data received.</dd>
+<dt markdown="1">**Clear**</dt>
+<dd>All collected data plotted in charts and event timeline are deleted and all snapshots, analyses and allocation monitoring is deleted too.</dd>
+<dt markdown="1">**Display dropdown**</dt>
 <dd>All memory data collected (timeseries).
-    The left-most or first time/data (memory information),
-    in the selector is when the application was launched.
-    The right-most or last time/data is the continual memory
-    information being received (live) until the application is
-    stopped.</dd>
-<dt markdown="1">**Range selector view**</dt>
-<dd>Detailed view of the data collected
-    for this timeseries range (non-gray area).</dd>
-<dt markdown="1">**X-axis timestamp**</dt>
-<dd>Time of the collected memory information
-    (capacity, used, external, RSS (resident set size),
-    and GC (garbage collection).</dd>
-<dt markdown="1">**Hover information of collected data**</dt>
-<dd>At a particular time (x-axis) the detailed collected
-    memory data.</dd>
-<dt markdown="1">**Garbage collection occurred**</dt>
-<dd>Compaction of the heap occurred.</dd>
-<dt markdown="1">**Event timeline**</dt>
-<dd>When a user action occurred (such as the
-    Snapshot or Reset button being clicked).</dd>
+    The left-most or first time/data (memory information) when the application was launched. The right-most or last time/data is the continual memory information received (live) until the application is stopped.  The timeline is scaled by the selected "Display" dropdown e.g., 
+<dt markdown="1">- Display 1 Minute</dt>
+<dt markdown="1">- Display 5 Minutes</dt>
+<dt markdown="1">- Display 10 Minutes</dt>
+<dt markdown="1">- Display All Minutes (slider disabled)</</dt>
+</dd>
+<dt markdown="1">**Source**</dt>
+<dd>Source can be either "Live Feed" current connected running Flutter application or offline "Export" file.</dd>
```

Comment:
Source can either be "Live Feed", which pulls data from the connected Flutter app, or one of the available offline data files, which are created by clicking "Export".

Reply:
Done.